### PR TITLE
Enhance audit log message for Group Perm updates

### PR DESCRIPTION
**Pull request by bosvos**
Include the actual permission names added / removed for the group in the audit message.

### CODE REVIEW

File: auth/http.go [hunk 1]
```diff
@@ -1098,6 +1117,23 @@ func httpHandlerSetGroupRoles(central *ImqsCentral, w http.ResponseWriter, r *ht
 	broadcastGroupChange(central, r, groupname)
 }
 
+func Diff(a authaus.PermissionList, b authaus.PermissionList) authaus.PermissionList {
```
Comment (FritzOnFire) [inline auth/http.go:1120]: Rather make the `Diff` a function on `authaus.PermissionList`, incase we one day want to create another Diff function (silly go an its lack of method overloading)

All thou this means that you would have to make changes in authaus I believe

**Review by FritzOnFire — CHANGES_REQUESTED** [review]

File: auth/http.go (hunk 1 above)
Comment (bosvos) [reply auth/http.go:1120]: ok will do

**Comment by bosvos** [comment]
@FritzOnFire changes pushed for imqs auth. See temporary 'replace' directive in go.mod

**Review by FritzOnFire — APPROVED** [review]
LGTM

**Pull request closed without merging**